cloudbuild-tag.yaml

```diff
@@ -0,0 +1,12 @@
+steps:
+
+- name: gcr.io/cloud-builders/docker
```

Comment:
Optionally - I would suggest to split this for two steps. Similar to https://github.com/GoogleCloudPlatform/marketplace-vm-imagebuilder/blob/master/cloudbuild.yaml#L37-L49.

Reply:
Done.